modules/ingest-grok/src/test/resources/rest-api-spec/test/ingest_grok/10_basic.yaml

```diff
@@ -0,0 +1,12 @@
+"Ingest grok installed":
+    - do:
+        cluster.state: {}
+
+    # Get master node id
+    - set: { master_node: master }
```

Comment:
In the ingest module we use the cluster _stats_ api and allows us easily to check whether a plugin is installed, but the cluster stats api doesn't return modules... (which we should add!)

So I had to do what other modules do too here. First use the cluster state api to get the id of an node and then use the node info api to get the specific modules information from that specific node (node info api return info per node id). In the cluster state api response, master is a field that is always around and returns the node id of the elected master node.